apps/dotcms-ui/src/app/api/services/dot-format-date-service.ts

```diff
@@ -27,7 +27,14 @@ export class DotFormatDateService {
     }
 
     async setLang(lang: string) {
-        const localeLang = await import(`date-fns/locale/${lang}/index.js`);
+        const customLangPath = {
```

Comment:
so basically, when the app lang is set to English, the variable `lang` has the value of `en`; but the path in `/node_modules/Date-fns/Locale/en/...` does not exist!

the one that exist is `/node_modules/Date-fns/Locale/en-US/...`, so a custom path needs to be created for those 4 languages

Reply:
This can't be static. We have the languages of the dotcms in the `appconfiguration` response:

![image](https://user-images.githubusercontent.com/751424/133338719-bd279999-4a3b-41b1-9aa6-dc3114a2eaa2.png)